adapters/beintoo/usersync.go

```diff
@@ -0,0 +1,12 @@
+package beintoo
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewBeintooSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("Beintoo", 457, temp, adapters.SyncTypeIframe)
```

Comment:
Is the 457 id correct? I can't find it included in the GDPR Global Vendor List, either version 1 or 2.